templates/case-study/index.html

```diff
@@ -0,0 +1,28 @@
+{% set hide_wrapper = True %}
+{% extends "case-study/base_case-study.html" %}
+
+{% block title %}Case Studies{% endblock %}
+{% block meta_description %}Ubuntu Case Studies{% endblock %}
+
+<!-- Set resource_types and tags somewhere -->
+{% block content %}
+<section class="p-strip--suru-blog-header is-dark">
+  <div class="row">
+    <div class="col-6">
+      <h1>Case Studies</h1>
+      <p>Ubuntu case studies.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-shallow" id="posts-list">
+  <div class="row u-equal-height">
+    {% for case_study in case_study_info %}
```

Comment:
what sorting will be applied here?

Reply:
Ideally it would be sorted by most recent date published. Moving forward, each individual case study should have a timestamp of the date of creation, and this timestamp can be used for the sorting from `views.py`.